supportedBackends/htcondor/src/test/scala/cromwell/backend/impl/htcondor/HtCondorRuntimeAttributesSpec.scala

```diff
@@ -251,4 +255,20 @@ class HtCondorRuntimeAttributesSpec extends WordSpecLike with Matchers {
       case ex: RuntimeException => assert(ex.getMessage.contains(exMsg))
     }
   }
+
+  private def createRuntimeAttributes(wdlSource: WdlSource, runtimeAttributes: String): Seq[Map[String, WdlValue]] = {
```

Comment:
IMO this is still unnecessarily circuitous. Why can't we just be explicit about the Seq[Map[String, WdlValue]] that we actually want and write it down in the tests?